src/steps/measureStep.ts

```diff
@@ -20,6 +20,12 @@ export const measureStep = (
     STEP_THRESHOLDS[config.steps[step].threshold];
 
   const stepMeasure = WP.measure(stepMetricName, M + startMark, M + endMark);
+
+  // checking to ensure stepMeasure is defined - it can be undefined if measure is called on a mark that has already been measured and cleared
```

Comment:
when write comment, make sure to not have them too long

so instead of
```
// checking to ensure stepMeasure is defined - it can be undefined if measure is called on a mark that has already been measured and cleared
```

do

```
// checking to ensure stepMeasure is defined - it can be undefined 
// if measure is called on a mark that has already been measured and cleared
```

it's easier to read